math/mean_absolute_percentage_error.cpp

```diff
@@ -0,0 +1,46 @@
+#include<iostream>
+#include<cmath>
+#include <vector>
+double mean_absolute_percentage_error(const std::vector<float>data_predict,const std::vector<float>data_actual){
```

Comment:
pakai tanda ampersand
```suggestion
double mean_absolute_percentage_error(const std::vector<float>& data_predict, const std::vector<float>& data_actual){
```